animeal.xcodeproj/project.pbxproj

```diff
@@ -1543,6 +1543,7 @@
 				INFOPLIST_FILE = animeal/res/Info.plist;
 				INFOPLIST_KEY_CFBundleDisplayName = "";
 				INFOPLIST_KEY_LSApplicationCategoryType = "";
+				INFOPLIST_KEY_NSCameraUsageDescription = "We use the camera so you can take pictures";
```

Comment:
Do we need this change in this PR?

Reply:
Yep I made it in on purpose

Reply:
Ok. I would expected to see NSLocationWhenInUseUsageDescription for such change because currently it looks like is not related to the PR so thought maybe better to separate one for clarity.

Reply:
got it. related to EPMEDU-605